Makefile

```diff
@@ -328,6 +338,14 @@ oracle8: ## Build Oracle Linux 8 image
 	-v ${VERBOSITY} \
 	$(if $(ADDITIONAL_OVERRIDES),--overrides=${ADDITIONAL_OVERRIDES})
 
+.PHONY: provision
+provision: build
+provision:
+	./bin/konvoy-image provision --inventory-file $(INVENTORY_FILE)  \
+	-v ${VERBOSITY} \
+	$(if $(ADDITIONAL_OVERRIDES),--overrides=${ADDITIONAL_OVERRIDES}) \
+	$(if $(EXTRA_OVERRIDE_VARS), --extra-vars=${EXTRA_OVERRIDE_VARS})
+
```

Comment:
How is this used?

Reply:
This is not used in automation yet. I was using it for testing now.  `make devkit.run WHAT="make provision`. however i kept it here to think that we can use it for testing `pre-provisioned` environment later.